theme/themes/pastanaga/extras/main.variables

```diff
@@ -11,3 +11,8 @@
 /* Mobile Menu Nav Item */
 @mobileMenuNavItemJustifyContent: center;
 @mobileMenuNavItemLineHeight: 1.25;
+
+/* Checkbox */
+@checkboxBorderUnchecked: 1px solid #826a6a;
```

Comment:
@sneridagh Any better suggestion for color? I copied this from pastanaga designs mockups from zeplin.

Reply:
Pastanaga is ok, since it's the old stuff. In your screenshot is it possible that it has a shadow? 
I can't see any in Pastanaga mocks. Could you please check?

<img width="218" alt="image" src="https://user-images.githubusercontent.com/486927/140641417-c92ca9fd-13fe-44af-ae72-8fb8d304b8e6.png">

Reply:
@sneridagh Done! Also I think shadows looks best with lighter borders🤔?